test/fit_tests/tests/rackhd20/test_rackhd20_api_rack_node.py

```diff
@@ -125,53 +96,60 @@ def tear_down(self):
 
     def test_api_create_and_check_racks(self):
         for operator in range(0,257):
-            RandomCharacters = '.'.join(random.sample(string.printable,8))
-            rackname = "myRack" + "_" + " " + RandomCharacters + datetime.now().__str__() + " " + RandomCharacters
+            if fit_common.VERBOSITY >= 8:
+                RandomCharacters = '.'.join(random.sample(string.printable,8))
+                rackname = "myRack" + "_" + " " + RandomCharacters + datetime.now().__str__() + " " + RandomCharacters
+            else:
+                RandomCharacters = '.'.join(random.sample(string.letters,8))
+                rackname = "myRack" + "_" + " " + RandomCharacters  + " " + RandomCharacters
             Newrack = {"name" :rackname,"type":"rack"}
             print "create new tag: " ,Newrack
-            mon_url = '/api/1.1/nodes'
+            mon_url = '/api/2.0/nodes'
             mon_data = fit_common.rackhdapi(mon_url,action='post',payload=Newrack)
             self.assertIn(mon_data['status'],range(200,205),"Incorrect HTTP return code: {}".format(mon_data['status']))
             rackid = mon_data['json']['id']
-            mon_url = '/api/1.1/nodes/{}'.format(rackid)
+            mon_url = '/api/2.0/nodes/{}'.format(rackid)
             mon_data = fit_common.rackhdapi(mon_url)
             self.assertIn(mon_data['status'],range(200,205),"Incorrect HTTP return code: {}".format(mon_data['status']))
             json_node_data = mon_data['json']
             self.assertTrue(json_node_data['name'] == Newrack['name'] and json_node_data['type']=="rack","rack node field error")
             print "query rack: " + rackname + "successfully!"
-    print "test: rack creation and query succeed!"
+        print "test: rack creation and query succeed!"

```

Comment:
this is the only test you are printing 'success' comments.  I'd remove the prints from this test, or add to all the rest of the tests to keep it consistent.